### PR TITLE
To discuss: GitHub Issue and PR templates

### PR DESCRIPTION
**Pull request by pieterlexis**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+ - Software source: <!-- e.g. Operating system repository, PowerDNS repository, compiled yourself -->
+
+### Steps to reproduce
+<!-- Tell us step-by-step how the issue can be triggered. Please include your configuration file and any (Lua) scripts that are loaded. -->
+1. <!-- step 1 -->
+2. <!-- step 2 -->
+3. <!-- step 3 -->
+
+### Expected behaviour
+<!-- What would you expect what happens when the reproduction steps are run -->
```
Comment (rgacogne) [inline .github/ISSUE_TEMPLATE.md]: > What would you expect what happens [...]

What would you expect to happen [...] ?

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+<!-- Before filing an issue, please search the existing issues (both open and closed) to see if your report might be duplicate -->
+<!-- Please don't file an issue when you have a support question, send support questions to the mailinglist (https://www.powerdns.com/mailing-lists.html) -->
```
Comment (Habbie) [inline .github/ISSUE_TEMPLATE.md]: maybe mention IRC here, i.e. link to https://www.powerdns.com/opensource.html instead?

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+ - Issue type: Bug report/Feature request <!-- delete the one that does not apply -->
+
+### Short description
+<!-- Tell in a few words what the issue/request is -->
+
+
+
+
+<!--
+If this is a bug report, use the following part the the template and delete the part a the bottom
```
Comment (Habbie) [inline .github/ISSUE_TEMPLATE.md]: part of, at the bottom

**Comment by pieterlexis** [comment]
Fixed nits!

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
+<!-- Before filing an issue, please search the existing issues (both open and closed) to see if your report might be duplicate -->
+<!-- Please don't file an issue when you have a support question, send support questions to the mailinglist or ask them on IRC (https://www.powerdns.com/opensource.html) -->
```
Comment (Habbie) [inline .github/ISSUE_TEMPLATE.md]: link to mailinglists?

File: .github/ISSUE_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+<!-- Tell us what is issue is about -->
+ - Program: Authoritative, Recursor, dnsdist <!-- delete the ones that don't apply -->
```
Comment (Habbie) [inline .github/ISSUE_TEMPLATE.md]: do not

File: .github/ISSUE_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+ - Issue type: Bug report/Feature request <!-- delete the one that does not apply -->
+
+### Short description
+<!-- Tell in a few words what the issue/request is -->
```
Comment (Habbie) [inline .github/ISSUE_TEMPLATE.md]: 'Explain in a few sentences'

File: .github/ISSUE_TEMPLATE.md (hunk 4 above)
Comment (pieterlexis) [inline .github/ISSUE_TEMPLATE.md]: that link has the links to the mailing list

Comment (Habbie) [inline .github/ISSUE_TEMPLATE.md]: Oh duh.

**Pull request merged**